project/project/MetaBuildShared.scala

```diff
@@ -0,0 +1 @@
+../MetaBuildShared.scala
```

Comment:
Q) Why do this file is needed? 
It appears that `scalajsJsEnvsVersion` is not used in  `project/project/~`.

Reply:
It is because `project/project/*.scala` files are available to `project/*.sbt` files as `project/*.scala` files are available to `*.sbt` files.

Reply:
I could also system-link it to `project/MetaBuildShared.sbt` which should be equivalent to `project/project/MetaBuildShared.scala`

Reply:
@exoego I can also skip this DRY attempt and have the version defined twice.
If you can, let me know what you prefer so we can merge this PR :)

Reply:
I still do not get it.
Again, it appears that scalajsJsEnvsVersion is not used in project/project/~.
So I prefer not having unused file.
Is scalajsJsEnvsVersion used somewhere in project/project/~ ??

Reply:
`project/project/*.scala` files are compiled together with `project/*.sbt` so it is used in `project/plugins.sbt`.

It it's the same situation when you define a `project/Build.scala`, you might think that it is unused in `project/~` but the values are used in root `build.sbt`

Reply:
If I delete `project/project/MetaBuildShared.scala` I get:
```scala
[info] welcome to sbt 1.5.3 (Homebrew Java 11.0.18)
/Users/lorenzo/scala/scala-js-env-jsdom-nodejs/project/plugins.sbt:6: error: not found: value MetaBuildShared
libraryDependencies += "org.scala-js" %% "scalajs-env-nodejs" % MetaBuildShared.scalajsJsEnvsVersion
                                                                ^
[error] Type error in expression

```

Reply:
Ah now I see.
Symlink is fine.